draft-ietf-taps-interface.md

```diff
@@ -887,12 +887,27 @@ over these options.
 Name:
 : multipath
 
-This property specifies whether an application considers it useful to
-transfer data across multiple paths between the same end hosts. Generally,
-in most cases, this will improve performance (e.g., achieve greater throughput).
-One possible side-effect is increased jitter, which may be problematic for
-delay-sensitive applications. The recommended default is to Ignore this option.
+Type:
+: Enumeration
+
+This property specifies whether an application wants to take advantage of
+transferring data across multiple paths between the same end hosts. Using
+multiple paths allows connections to migrate between interfaces as
+availability and performance properties change. Possible values are:
+
+Disabled:
+: The connection will not support using multiple paths once established
+
+Handover:
+: The connection will support migrating between different paths upon interface availability changes
+
+Interactive:
+: The connection will support using multiple paths in response to loss or delay upon individual paths
+
+Aggregate:
+: The connection will support using multiple paths in parallel in order to maximize bandwidth
```

Comment:
"will" makes these choices sound like promises, when they are only expressing a wish by the application that may or may not be granted. I'd like to request changing the three instances of "The connection will support ..."  above into "The connection should support ...". The only promise we can keep is the Disabled choice, "will not support" is fine.

Reply:
Good call, fixed! Please take a look.

Reply:
Thanks for fixing this!